src/pages/Checkout/CheckoutPage.vue

```diff
@@ -502,6 +504,21 @@ export default {
 			);
 		}
 
+		// MARS-452 tip rate optimization experiment
+		const tipRateOptimizationExperiment = this.apollo.readFragment({
+			id: `Experiment:${TIP_RATE_OPTIMIZATION_EXP}`,
+			fragment: experimentVersionFragment,
+		}) || {};
+
+		this.tipRateOptimizationExperimentVersion = tipRateOptimizationExperiment.version;
+		if (this.tipRateOptimizationExperimentVersion && !this.checkingOutAsGuest) {
```

Comment:
If guest users are to be excluded, please wrap all of this in `!this.checkingOutAsGuest` instead of just the tracking.